Library/Formula/cfssl.rb

```diff
@@ -0,0 +1,103 @@
+require "language/go"
+
+class Cfssl < Formula
+  desc "CloudFlare's PKI toolkit"
+  homepage "https://cfssl.org/"
+  url "https://github.com/cloudflare/cfssl/archive/1.1.0.tar.gz"
+  sha256 "589222d922aa4ebae2039a54c46b9a7c2df4c95d78026ab35768a6cf62836ca9"
+  head "https://github.com/cloudflare/cfssl.git"
+
+  bottle do
+    cellar :any_skip_relocation
+  end
+
+  option "with-pkcs11", "Enables PKCS11 support and adds dependency on libtool"
```

Comment:
Is there any reason to provide this as an option instead of using it by default?